modular_skyrat/master_files/code/datums/traits/neutral.dm

```diff
@@ -114,3 +114,12 @@
 				H.emote("laugh")
 				addtimer(CALLBACK(H, /mob/proc/emote, "laugh"), 5 SECONDS)
 				addtimer(CALLBACK(H, /mob/proc/emote, "laugh"), 10 SECONDS)
+
+/datum/quirk/feline_aspect
+	name = "Feline Traits"
+	desc = "You happen to act like a feline, for whatever reason."
+	gain_text = "<span class='notice'>Nya~ could go for some catnip wight about now...~ OwO</span>"
```

Comment:
That's just really ugly, find something less "uwu" please.

Reply:
![angy-bladerunner](https://user-images.githubusercontent.com/77863456/129139539-d9781ede-2190-4952-b2cc-a2c48da5c05e.gif)